requirements.txt

```diff
@@ -2,7 +2,7 @@ pyyaml >= 5.1
 astropy >= 4.0
 click >= 7.0
 sqlalchemy >= 1.3
-https://github.com/lsst/sphgeom/archive/master.tar.gz
+git+git://github.com/lsst/sphgeom@master#egg=lsst_sphgeom
```

Comment:
why is this better than the previous URL?

Reply:
I spent some time trying to find documentation about this. The following is the best link I found: https://pip.pypa.io/en/stable/reference/pip_install/#requirement-specifiers

and it pointed me to the following for the various git protocols: https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols
If I read this correctly I think the main advantage to `git+git` is speed. It can only be used for unauthenticated read, which is fine for this use case.